server/routes/money.js

```diff
@@ -38,6 +39,84 @@ const createMoneyRouter = ({ hubContentService, offenderService }) => {
     }
   });
 
+  router.get('/transactions', async (req, res, next) => {
+    try {
+      const config = {
+        content: false,
+        header: false,
+        postscript: true,
+        detailsType: 'small',
+        returnUrl: req.originalUrl,
+      };
+
+      const { user } = req;
+
+      const data = {};
+
+      if (user) {
+        const accountTypes = ['spends', 'private', 'savings'];
+        const accountType = accountTypes.includes(req.query.accountType)
+          ? req.query.accountType
+          : accountTypes[0];
+        const accountTypeMachineName = {
+          spends: 'spends',
+          private: 'cash',
+          savings: 'savings',
+        };
+
+        const transactionsTo = new Date();
+        const transactionsFrom = subDays(transactionsTo, 30);
+
+        const [transactions, balances] = await Promise.all([
+          offenderService.getTransactionsFor(
+            user,
+            accountTypeMachineName[accountType],
+            formatISO(transactionsFrom, { representation: 'date' }),
+            formatISO(transactionsTo, { representation: 'date' }),
+          ),
+          offenderService.getBalancesFor(user),
+        ]);
+
+        const listOfPrisons = Array.from(
+          new Set(transactions.map(transaction => transaction.prison)),
+        );
```

Comment:
This looks nifty. What does it do?

Reply:
This has been removed following review from Andy bellow, it was to get a unique list of prisons from the transactions in order to query the Prison API, it was no longer required following the API change